DEPS

```diff
@@ -31,7 +31,7 @@ vars = {
   'cassowary_dart_revision': '7e5afc5b3956a18636d5b37b1dcba1705865564b',
   'collection_dart_revision': '79ebc6fc2dae581cb23ad50a5c600c1b7dd132f8',
   'crypto_dart_revision': 'd4558dea1639e5ad2a41d045265b8ece270c2d90',
-  'newton_dart_revision': '9fbe5fbac809246f7ace4176feca13bdf731e393',
+  'newton_dart_revision': '11cd659e1650591402b69f91b9c4d2a0841fd5bd',
```

Comment:
Can you update //sky/sdk/pubspec.yaml as well so that when we release this code pub will pull in the correct version?